backend/question-service/models/Category.ts

```diff
@@ -1,10 +1,7 @@
 import mongoose, { Schema, Types, Document } from 'mongoose';
-import { Category as SharedCategory } from '@shared/types/Category';
-
-export interface Category
-  extends Omit<SharedCategory, '_id'>,
-    Document {
+export interface Category extends Document {
   _id: Types.ObjectId;
+  name: string;
 }
```

Comment:
I think the former method of extending and omitting from SharedCategory in alignment with CS3219 priorities of maintainability and scalability. If anything, Question.js should be changed to match this implementation.